opts.py

```diff
@@ -227,6 +227,8 @@ def train_opts(parser):
 def translate_opts(parser):
     parser.add_argument('-model', required=True,
                         help='Path to model .pt file')
+    parser.add_argument('-model_type', default='text',
+                        help="Type of encoder to use. Options are [text|img].")
```

Comment:
We should save this with the file. If you need an argument here it should be `data_type`